third_party/move/move-model/tests/sources/structs_err.v2_exp

```diff
@@ -22,11 +22,11 @@ error: missing fields `y`, `z`
 40 │       S{x: x}
    │       ^^^^^^^
 
-error: cannot return `G<u64>` from a function with result type `G<bool>`
```

Comment:
These changes are unrelated to this PR, but were obtained by running `move_pr.sh -i2`.